api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -7,10 +7,355 @@ type: Object
 Configures client-side exporting.
 
 ---
-When client-side exporting is [enabled](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/export/enabled.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/#enabled'), the grid toolbar contains the Export button ( <img src="/Content/images/doc/20_1/DataGrid/icons/toolbar_export.png" alt="DevExtreme DataGrid HTML5 Toolbar Exporting" style="vertical-align:middle"/> ) that exports grid data to Excel. For details on exporting, refer to the [Client-Side Exporting](/concepts/05%20Widgets/DataGrid/70%20Client-Side%20Exporting '/Documentation/Guide/Widgets/DataGrid/Client-Side_Exporting/') article.
+A user can click the **Export** button to save an Excel file with the exported data. Data types, sorting, filtering, and grouping settings are maintained.
 
-[note]Client-side exporting requires the JSZip library. Learn where you can get it from topics in the [Installation](/concepts/Common/Distribution%20Channels/15%20ZIP%20Archive.md '/Documentation/Guide/Common/Distribution_Channels/ZIP_Archive/') section.
+![DevExtreme HTML5 JavaScript DataGrid Export Button](/images/DataGrid/exported_data.png)
 
-#include common-demobutton with {
-    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/ExportToExcel/"
-}
\ No newline at end of file
+The following instructions show how to enable and configure client-side export:
+
+1. **Prerequisites and Restrictions**  
```

Comment:
The first step should be about the libraries only. Move the restrictions to the end. It also seems that we have to give a code example after all because it's not clear when we explain it with words.